torch/_subclasses/functional_tensor.py

```diff
@@ -205,11 +237,19 @@ def unwrap(x):
             torch._C.DispatchKey.Functionalize
         )
         assert is_excluded or not is_included
+        include_to_set = (
+            torch._C._dispatch_tls_local_include_set()
+            | torch._C.DispatchKeySet(torch._C.DispatchKey.Functionalize)
+        )
+        exclude_to_set = (
+            torch._C._dispatch_tls_local_exclude_set().remove(
+                torch._C.DispatchKey.Functionalize
+            )
+            - FunctionalTensor._extra_dispatch_keys
+        )
```

Comment:
NB: your PR message is saying that we did this in PythonFallBackKernel.cpp, but we're actually manually doing it in the subclass :P

Reply:
sigh totally forgot to update the PR description, will update